tutorials/packaging/create-your-first-snap/create-your-first-snap.md

```diff
@@ -63,14 +63,24 @@ This tutorial has been written to work with Ubuntu 16.04 LTS (Xenial Xerus) and
 
 ### Installing dependencies
 
-We will now update the APT database and install snapcraft and some additional build tools we are
-going to need:
+We will now update the APT database and install some additional build tools we are going to need:
 
 ```bash
 sudo apt update
-sudo apt install snapcraft build-essential
+sudo apt install build-essential
 ```
 
+We will now install snapcraft, the utility for building snaps:
+
+```bash
+sudo snap install --classic snapcraft
+```
+
+positive
+: **NOTE:**
+    * If the `snap` command is not available, install the `snapd` package via APT.
+    * The `--classic` switch enables the installation of a snap that is in *classic confinement*, we will discuss about snap's security confinement later in the following section.
```

Comment:
I'd just reword this slightly: 
The `--classic` switch enables the installation of a snap that uses *classic confinement*. We discuss snap security confinement in the following section.